src/tools/gud.md

```diff
@@ -0,0 +1,170 @@
+# Mozilla Growth & Usage Dashboard
+#### *overall: jmccrosky@ | data: jklukas@ | frontend: hulmer@ (updated 2020-1-15)*
+___
+
+**The Mozilla Growth & Usage Dashboard is in general release and can be found [here](https://growth-stage.bespoke.nonprod.dataops.mozgcp.net/).**
+
+___
+
+# Overview
+
+The Mozilla Growth & Usage Dashboard (GUD) is a tool to visualize growth metrics in a standard way across Mozilla’s products.  It is intended to be quite intuitive to use, but we provide some brief documentation here.  Please reach out (`#gud` on slack) if anything is unclear or you are unsure how to use GUD for your purposes.
+
+# Quick Start
+
+To visualize a particular metric (such as `MAU` or `Retention`) for a particular product:
```

Comment:
Most of the terms in backticks in this article probably don't need them, or should be italicized with underscores instead. In particular, MAU and retention both seem like natural words that don't need any adornment.

Reply:
My intent was to call out words that correspond to UI elements in the dashboard, like the PRODUCT / USAGE CRITERIA selector, or specific menu items like metrics and such.  Do you think this is not helpful?  Or helpful but better done with italics or something?

Reply:
I see your point. Let's roll with it.